win32_event_log/README.md

```diff
@@ -15,9 +15,11 @@ The Windows Event Log check is included in the [Datadog Agent][1] package. There
 
 ### Configuration
 
-1. Edit the `win32_event_log.d/conf.yaml` in the `conf.d/` folder at the root of your [Agent's configuration directory][2]. See the [sample win32_event_log.d/conf.yaml][3] for all available configuration options.
+1. First ensure that you have set `logs_enabled: true` in your `datadog.yaml` file.
```

Comment:
Let's move this under ```### Log collection` section instead. This would be confusing logs configuration and general configuration (Since logs is not required to make this check work)

Reply:
Thanks @ChristineTChen , I have made the recommended changes